cypher/cypher-docs/src/docs/dev/syntax/comparison.asciidoc

```diff
@@ -63,7 +63,26 @@ The following points give some details on how the comparison is performed.
  A leap year occurs on a year with an ordinal number divisible by `4`, that is not divisible by `100`, unless it divisible by `400`.
  This means that over `400` years there are `((365 * 4 + 1) * 25 - 1) * 4 + 1 = 146097` days, which means an average of `365.2425` days per year.].
 * Comparing for ordering when one argument is `null` (e.g. `null < 3` is `null`).
-
+* *Ordering* of values with *different* types:
+ ** The ordering is, in ascending order, defined according to the following list:
+  *** `MAP` types
+   **** <<cypher-literal-maps, Regular maps>>
+   **** <<cypher-map-projection, `NODE` maps>>
+   **** <<cypher-map-projection, `RELATIONSHIP` maps>>
+  *** <<cypher-lists, `LIST`>>
```

Comment:
We don't seem to use UPPERCASE for our types in any other place, But rather CamelCase, e.g. here: https://neo4j.com/docs/cypher-manual/current/syntax/temporal/#cypher-temporal-introduction